### PR TITLE
Fix: Changelog

### LINKED ISSUE #799: 500 server error on changelog page
https://webhint.io/about/changelog/ content is not available.

> The page cannot be displayed because an internal server error has occurred.

Accessed on Windows 10, Edge Canary 78.0.254.0. cc @codepo8.

### PR DESCRIPTION
**Pull request by sarvaje**
Fix #799

## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/webhint.io)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

### CODE REVIEW

File: helpers/update-site/documentation.js [hunk 1]
```diff
@@ -724,7 +724,7 @@ const createHintCategories = (files) => {
 };
 
 const updateChangelog = async () => {
-    const files = await globby([`${constants.dirs.HINT_PACKAGES}/*/CHANGELOG.md`]);
+    const files = await globby([`${constants.dirs.NODE_MODULES}/{hint,@hint/{${resources.join(',')},configuration-*/node_modules/@hint/{${resources.join(',')}}}-*}/CHANGELOG.md`]);
```
[the remark this answers is not in the document]
Comment (sarvaje) [reply helpers/update-site/documentation.js]: I think we weren't including configuration to the documentation, but I think we should include it.

[the remark this answers is not in the document]
Comment (sarvaje) [reply helpers/update-site/documentation.js]: > versus grabbing all @hint/*

I think we don't want to grab `@hint/utils*`.

[the remark this answers is not in the document]
Comment (antross) [reply helpers/update-site/documentation.js]: Good point. Though perhaps we should just be getting the list from `configuration-all`? That should already be excluding the packages we don't want. Then we're not duplicating the filtering logic here.

[the remark this answers is not in the document]
Comment (sarvaje) [reply helpers/update-site/documentation.js]: yep, I'm working on the `configuration-all` thing.

**Pull request merged**